scribble-lib/scribble/acmart/acmart.tex

```diff
@@ -4,8 +4,10 @@
 % These are replaced by scribble/acmart/style.tex,
 %  which is used in combination with acmart.cls
 
+\newcommand{\SAuthorinfo}[4]{#1}
```

Comment:
Ah, okay, yes. They do. The output is less important, but they should take 4 arguments to match the actual signature of the command. Although honestly I mostly just did that to match the convention of other scribble languages. Namely, check `sigplan/sigplan.tex`.